core/src/test/kotlin/io/sharing/server/core/product/domain/ProductTest.kt

```diff
@@ -50,6 +53,52 @@ class ProductTest {
             "image6", "image7", "image8", "image9", "image10"
         )
     }
+
+    @Test
+    fun `등록된 상품을 승인을 할 수 있다`() {
```

Comment:
실패케이스 조건 혹은 특이 케이스 (상품 등록할 수 있는 방법이 여러개 존재) 가 아니라면 구체적인 조건이 테스트명에 드러날 필요는 없어보입니다.

```suggestion
    fun `상품 승인 성공`() {
```